modules/workflow-service-impl/src/main/java/org/opencastproject/workflow/impl/WorkflowServiceImpl.java

```diff
@@ -1135,15 +1135,15 @@ protected void assertPermission(WorkflowInstance workflow, String action, String
   }
 
   protected boolean assertMediaPackagePermission(String mediaPackageId, String action) {
-    Opt<MediaPackage> assetMediaPackage = assetManager.getMediaPackage(mediaPackageId);
-    if (assetMediaPackage.isSome()) {
-      var currentUser = securityService.getUser();
-      var mediaPackage = assetMediaPackage.get();
```

Comment:
I was trying fixing this method, but now I know that the problem was mainly that line. It will fail with packages that doesn't have media package because there wasn't any.  I was not bold enough to delete it (And also I didn't understand the code completely).